src/goto-instrument/Makefile

```diff
@@ -1,4 +1,5 @@
-SRC = accelerate/accelerate.cpp \
+SRC = abstraction.cpp \
```

Comment:
Again, too generic, `replication_reducing_abstraction.cpp` ?

Reply:
Renamed to rr-abstraction.cpp.